lib/marc/controlfield.rb

```diff
@@ -2,15 +2,15 @@
 
 module MARC
 
-  # MARC records contain control fields, each of which has a 
+  # MARC records contain control fields, each of which has a
   # tag and value. Tags for control fields must be in the
   # 001-009 range or be specially added to the @@control_tags Set
 
   class ControlField
-    
+
     # Initially, control tags are the numbers 1 through 9 or the string '000'
     @@control_tags = Set.new(%w{000 001 002 003 004 005 006 007 008 009})
```

Comment:
Actually, in a latter comment I suggest this might be replaced entirely with a regex test in the `control_tag?` method and that method might raise the exception.